code/modules/vore/eating/bellymodes.dm

```diff
@@ -71,18 +72,9 @@
 
 		for (var/mob/living/M in contents)
 			if(prob(25))
-				if((world.time + NORMIE_HEARCHECK) > last_hearcheck)
-					LAZYCLEARLIST(hearing_mobs)
-					for(var/mob/living/H in get_hearers_in_view(3, owner))
-						if(!H.client || !(H.client.prefs.cit_toggles & DIGESTION_NOISES))
-							continue
-						LAZYADD(hearing_mobs, H)
-					last_hearcheck = world.time
-				for(var/mob/living/H in hearing_mobs)
-					if(H && H.client && (isturf(H.loc) || (H.loc != src.contents)))
-						SEND_SOUND(H,pred_digest)
-					else if(H && H in contents && H.client)
-						SEND_SOUND(H,prey_digest)
+				if(M && M.client && M.client.prefs.cit_toggles & DIGESTION_NOISES)
```

Comment:
So the world.time hearcheck removal for "preys" is intended, right?

Reply:
Yes because it was simply tied into the predator's noises